client-lib/go/client/jvs_client.go

```diff
@@ -50,12 +59,50 @@ func NewJVSClient(ctx context.Context, config *JVSConfig) (*JVSClient, error) {
 	}
 
 	return &JVSClient{
-		config: config,
-		keys:   cached,
+		config:           config,
+		keys:             cached,
+		forbidBreakglass: config.ForbidBreakglass,
 	}, nil
 }
 
 // ValidateJWT takes a jwt string, converts it to a JWT, and validates the signature.
 func (j *JVSClient) ValidateJWT(jwtStr string) (*jwt.Token, error) {
+	// Handle Break-glass tokens.
+	if strings.HasSuffix(jwtStr, UnsignedPostfix) {
+		token, err := jwt.Parse([]byte(jwtStr), jwt.WithVerify(false))
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse jwt %s: %w", jwtStr, err)
+		}
+		valid, err := j.unsignedTokenValidAndAllowed(token)
+		if !valid {
+			return nil, fmt.Errorf("token unsigned and could not be validated: %w", err)
+		}
+		return &token, nil
+	}
 	return jvscrypto.ValidateJWT(j.keys, jwtStr)
 }
+
+func (j *JVSClient) unsignedTokenValidAndAllowed(token jwt.Token) (bool, error) {
+	if j.forbidBreakglass {
```

Comment:
You can do the following instead: 

```suggestion
	if j.config.ForbidBreakglass {
```